Makefile

```diff
@@ -103,6 +112,23 @@ $(DIAMOND): $(DIAMOND_TGZ)
 $(DIAMOND_DB): $(DIAMOND) $(SMALLFASTA)
 	./$(DIAMOND) makedb --db $(DIAMOND_DB) --in $(SMALLFASTA) --threads $(AVAILCPU) $(DIAMOND_MAKEDB_OPTIONS)
 
+$(SEQR_JAR):
+	wget $(SEQR_JAR_URL)
+
+$(SEQR_SRC): 
+	wget $(SEQR_SRC_URL)
+	tar xvf seqr-clojure-$(SEQR_VER) 
+
+$(SEQR_DB): $(SEQR_SRC) $(SEQR_JAR) $(SMALLFASTA)
+	java -jar $(SEQR_JAR) index $(SMALLFASTA) --in_format fasta -d $(SEQR_SRC)/testdata/solr 
+
+# SEQR
+# *May* require java 8
+# Currently multi-cpu is not supported with embedded seqr because it causes database lock. it would work with server seqr.
+# Currently seqr is multi-threaded by default, not sure if it actually helps or not.
```

Comment:
I need to do some benchmarking myself to find out how to actually benefit from multiple threads. I think the server version may benefit--but this is the embedded version.